src/Playwright.Tests.TestServer/wwwroot/drag-n-drop.html

```diff
@@ -1,40 +1,43 @@
 <style>
-div:not(.mouse-helper) {
-  margin: 0em;
-  padding: 2em;
-}
-#source {
-  color: blue;
-  border: 1px solid black;
-}
-#target {
-  border: 1px solid black;
-}
+    div:not(.mouse-helper) {
+        margin: 0em;
+        padding: 2em;
+    }
+
+    #source {
+        color: blue;
+        border: 1px solid black;
+    }
+
+    #target {
+        border: 1px solid black;
+    }
 </style>
 
 <script>
 
-function dragstart_handler(ev) {
-  ev.currentTarget.style.border = "dashed";
-  ev.dataTransfer.setData("text/plain", ev.target.id);
-}
+    function dragstart_handler(ev) {
+        ev.currentTarget.style.border = "dashed";
+        ev.dataTransfer.setData("text/plain", ev.target.id);
+    }
 
-function dragover_handler(ev) {
-  ev.preventDefault();
-}
+    function dragover_handler(ev) {
+        ev.preventDefault();
+    }
 
-function drop_handler(ev) {
-  console.log("Drop");
-  ev.preventDefault();
-  var data = ev.dataTransfer.getData("text");
-  ev.target.appendChild(document.getElementById(data));
-}
+    function drop_handler(ev) {
```

Comment:
Please do not reformat the file, they should be 1:1 as upstream.

Reply:
Looks like that got reformatted as it was pasted.

Reply:
(I usually curl the file or copy it via `cp`, otherwise VSC does also not-wanted formatting). Its still reformatted after the latest push.